metrics/grafana/tikv_details.dashboard.py

```diff
@@ -1857,6 +1857,16 @@ def RaftIO() -> RowPanel:
             metric="tikv_raftstore_apply_log_duration_seconds",
         )
     )
+    layout.row(
+        heatmap_panel_graph_panel_histogram_quantile_pairs(
+            heatmap_title="Raft Message Wait Flush Duration",
+            heatmap_description="The time consumed for Raft Message wait flush",
+            graph_title="99% Raft Message Wait Flush Duration",
```

Comment:
It's more than 99%, it includes 99.99% and avg too.

```suggestion
            graph_title="Raft Message Wait Flush Duration",
```

Also, could you update panels around this one? They have similar issues.

Reply:
done.